src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/NestedIfDepth/context/Test.java

```diff
@@ -0,0 +1,34 @@
+package TreeWalker.coding.NestedIfDepth;
+
+public class Test {
+    public void test(int a) {
```

Comment:
Please add one more if-block to this method which produces a violation, but the filter suppresses it, because it is not changed in a patch file.

Reply:
done